### ISSUE TITLE
Exporting brush masks as "Brush labels to PNG" throws stacktrace when using mixed annotations

### ISSUE THREAD

**Issue by thomasmarkus** [heartexlabs/label-studio#1416]
**Describe the bug**
When attempting to export data from brush labels *and* other types, such as an Eplise annotation, data can no longer be exported to PNG. It seems there is a hardcoded dependency on RLE-style data (which is fine for brushes), but isn't available for other annotation types and therefore breaks the export.

**To Reproduce**

Our labelling setup looks like this:

```<View>
  <Image name="image" valueType="url" value="$image" zoom="true" zoomControl="false" rotateControl="false"/>
  <Ellipse name="ellipse" toName="image"/>
  <BrushLabels name="tag" toName="image">
    <Label value="red" background="rgba(255, 0, 0, 0.6)"/>
    <Label value="yellow" background="rgba(255, 255, 0, 0.6)"/>
    <Label value="black" background="rgba(0, 0, 0, 0.6)"/>
    <Label value="other" background="rgba(0, 0, 255, 0.6)"/>
  </BrushLabels>
  <Header value="Can you annotate the wound in the image?"/>
  <Choices name="comparison" toName="image" showInline="false">
    <Choice value="yes"/>
    <Choice value="no"/>
  </Choices>
</View>
```

Annotate at least one image with a brush *and* elipse to trigger the issue.

**Expected behavior**
A zip with png labels, but got a stacktrace instead (see below)

**Stacktrace**

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/django/utils/decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "/label-studio/label_studio/data_export/api.py", line 149, in get
    export_stream, content_type, filename = DataExport.generate_export_file(
  File "/label-studio/label_studio/data_export/models.py", line 95, in generate_export_file
    converter.convert(input_json, tmp_dir, output_format, is_dir=False)
  File "/usr/local/lib/python3.8/dist-packages/label_studio_converter/converter.py", line 183, in convert
    brush.convert_task_dir(items, output_data, out_format='png')
  File "/usr/local/lib/python3.8/dist-packages/label_studio_converter/brush.py", line 154, in convert_task_dir
    convert_task(item, out_dir, out_format)
  File "/usr/local/lib/python3.8/dist-packages/label_studio_converter/brush.py", line 146, in convert_task
    save_brush_images_from_annotation(item['id'], item['annotation_id'], item['completed_by'],
  File "/usr/local/lib/python3.8/dist-packages/label_studio_converter/brush.py", line 124, in save_brush_images_from_annotation
    layers = decode_from_annotation(from_name, results)
  File "/usr/local/lib/python3.8/dist-packages/label_studio_converter/brush.py", line 106, in decode_from_annotation
    rle = result['rle']
KeyError: 'rle'
```

**Environment (please complete the following information):**
 - Label Studio Version 1.2.0

**Comment by makseq** [comment]
@thomasmarkus Could you switch label-studio-converter to the latest master, I hope this will work for you:
https://github.com/heartexlabs/label-studio-converter/commit/7cb76650456b753ad9d9abf8aecabca712535343

**Comment by thomasmarkus** [comment]
Seems fixed.
Validated with current https://github.com/heartexlabs/label-studio/commit/a9cd862ab4fe2cde17613a4a64f9cf1eeed583d7and manually updating the dependency with `pip install -U git+https://github.com/heartexlabs/label-studio-converter.git@7cb76650456b753ad9d9abf8aecabca712535343#label-studio-converter`

**Comment by makseq** [comment]
It will be released in LS 1.3.
